examples/jsm/nodes/shadernode/ShaderNode.js

```diff
@@ -311,6 +313,8 @@ export const nodeArray = ( val ) => new ShaderNodeArray( val );
 export const nodeProxy = ( ...val ) => new ShaderNodeProxy( ...val );
 export const nodeImmutable = ( ...val ) => new ShaderNodeImmutable( ...val );
 
+export const shader = ( ...val ) => new ShaderNode( ...val );
```

Comment:
Shouldn't we then change usages of `new ShaderNode` (for example, in Skinning) to `shader`?

Reply:
> Shouldn't we then change usages of new ShaderNode (for example, in Skinning) to shader?

I think so.